paperqa/settings.py

```diff
@@ -609,6 +618,10 @@ class Settings(BaseSettings):
             " router_kwargs key with router kwargs as values."
         ),
     )
+    use_batch_in_summary: bool = Field(
+        default=False,
+        description="Whether to use batch API for LLMs in summarization",
```

Comment:
Can you add a few words on how the batches are actually formed

Reply:
Perhaps you can say something like:

> Whether to use batch API for LLMs in summarization, which means multiple messages are sent in one API request.

Reply:
It was updated to:
```
"Whether to use batch API for LLMs in summarization, "
"which means multiple messages are sent in one API request "
"to the LLM provider's batch API."
"This option is only available for Claude(https://docs.anthropic.com/en/api/creating-message-batches)"
"and OpenAI (https://platform.openai.com/docs/guides/batch) chat models."
```